### DISCUSSION TITLE
TTS API token

### DISCUSSION THREAD

**Discussion by amdapsi** [nextai-translator/nextai-translator#964] [Q&A]
I am implementing an app with reference to OpenAI Translator.
Is it legally permissible to use the API key for Text-To-Speech?

https://github.com/openai-translator/openai-translator/blob/main/src/common/tts/edge-tts.ts#L33

**Comment by maltoze** [comment]
Yes, this token is actually from here https://github.com/rany2/edge-tts/blob/8f8a3342038a283917c39af11435adb729f4ce42/src/edge_tts/constants.py#L5
